action/kafkaProduce.py

```diff
@@ -1,4 +1,5 @@
 import ssl
+import base64
 from kafka import KafkaProducer
 from kafka.errors import NoBrokersAvailable
 
```

Comment:
@jtyberg, unfortunately the updates are not compatible with Python 2 due to lines like this `bytes(validatedParams['value'], 'utf-8')`. The `bytes` method only takes one argument in Python 2.